examples/Interceptor/Server/Services/GreeterService.cs

```diff
@@ -53,5 +53,27 @@ public override async Task SayHellos(HelloRequest request, IServerStreamWriter<H
                 await Task.Delay(1000);
             }
         }
+
+        public override async Task<HelloReply> SayHelloToLotsOfBuddies(IAsyncStreamReader<HelloRequest> requestStream, ServerCallContext context)
+        {
+            var message = "Hello";
+            await foreach (var request in requestStream.ReadAllAsync())
+            {
+                message += $" {request.Name},";
+            }
+
+            message = message.Remove(message.Length - 1);
```

Comment:
```suggestion
            var names = new List<string>();
            await foreach (var request in requestStream.ReadAllAsync())
            {
                names.Add(request.Name);
            }

            var message = $"Hello {string.Join(", ", names)}";
```

Reply:
Minor, but I like building up a collection of names and then generating the message.